site-src/guides/index.md

```diff
@@ -58,9 +63,46 @@ documentation](/concepts/versioning/).
 To install the experimental channel, run the following kubectl command:
 
 ```bash
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/experimental-install.yaml
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/experimental-install.yaml
 ```
 
+### v1.1 Upgrade Notes
+If you are already using previous versions of GRPCRoute or BackendTLSPolicy
+experimental channel CRDs from previous Gateway API releases, you'll need to be
+careful with this upgrade. If you haven't installed Gateway API before, or have
```

Comment:
`you` here is the user, but most users today dont install Gateway API CRDs, they rely on implementations to install Gateway API CRDs and in some cases like GKE where the cloud provider installs it

Reply:
Do you have any examples of implementations bundling CRDs? So far we've recommended that CRD management is owned by users or cluster providers (https://gateway-api.sigs.k8s.io/guides/crd-management/#who-should-manage-crds).